### PR TITLE
docs: try to improve script --input-file

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: docs/source/daml-script/index.rst [hunk 1]
```diff
@@ -212,7 +224,7 @@ We can then initialize our ledger passing in the json file via ``--input-file``.
 
 If you open Navigator, you can now see the contracts that have been created.
 
-While we will not use it here, there is also an ``--output-file``
+As mentioned, there is also an ``--output-file``
```
Comment (cocreature) [inline docs/source/daml-script/index.rst:227]: Can we fold this into the paragraph above? It seems a bit weird to mention the option above without explaining it and then mention it somewhere down here.

**Review by cocreature — APPROVED** [review]
Thanks!

File: docs/source/daml-script/index.rst (hunk 1 above)
Comment (garyverhaegen-da) [reply docs/source/daml-script/index.rst]: I've tried moving this up, which I think flows a little bit better. Thoughts?

Comment (cocreature) [reply docs/source/daml-script/index.rst]: I like it, thanks!

**Pull request merged**